docs/developer/tutorials/snmp/profiles.md

```diff
@@ -0,0 +1,156 @@
+# Build an SNMP profile
+
+SNMP profiles are our way of providing out-of-the-box monitoring for certain makes and models of network devices.
+
+This tutorial will walk you through the steps of building a basic SNMP profile that collects OID metrics from HP iLO4 devices.
+
+Feel free to read the [Introduction to SNMP](./introduction.md) if you need a refresher on SNMP concepts such as OIDs and MIBs.
+
+Ready? Let's get started!
+
+## Research
+
+The first step to building an SNMP profile is doing some basic research about the device, and which metrics we want to collect.
+
+### General device information
+
+Generally, you'll want to search the web and find out about the following:
+
+- Device name, manufacturer, and device `sysobjectid`.
+
+- Understand what the device does, and what it is used for. (Which metrics are relevant varies between routers, switches, bridges, etc. See [Networking hardware](https://en.wikipedia.org/wiki/Networking_hardware).)
+
+    > E.g. from the [HP iLO Wikipedia page](https://en.wikipedia.org/wiki/HP_Integrated_Lights-Out), we can see that iLO4 devices are used by system administrators for remote management of embedded servers.
+
+- Available versions of the device, and which ones we target.
+
+    > E.g. HP iLO devices exist in multiple versions (version 3, version 4...). Here, we are specifically targetting HP iLO4.
+
+- Supported MIBs and OIDs (often available in official documentation), and associated MIB files.
+
+    > E.g. we can see that HP provides a MIB package for iLO devices [here](https://support.hpe.com/hpsc/swd/public/detail?swItemId=MTX_53293d026fb147958b223069b6).
+
+### Metrics selections
```

Comment:
Can we include info on SNMP types -> integration types? What we support and what we don't?

Reply:
Good idea, added a section about this and also `forced_type` :)